src/demo/demo-controlled-inputs/App.css

```diff
@@ -16,18 +16,18 @@
   font-size: 16px;
 }
 
-article {
-  padding: 1rem;
-  box-shadow: 1px 2px 3px #ddd;
-}
+/*article {*/
```

Comment:
Pls remove outcommented code. If we need it in the future, we have version control to see how it was.

Reply:
fixes in #f027467